tests/foreman/cli/test_import.py

```diff
@@ -0,0 +1,110 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Host Collection CLI"""
+from ddt import ddt
+from fauxfactory import gen_string
+from robottelo.common import ssh
+from robottelo.common.helpers import prepare_import_data
+from robottelo.test import CLITestCase
+from robottelo.cli.import_ import Import
+from robottelo.cli.org import Org
+
+
+@ddt
+def csv_to_dataset(csv):
+    """Converts the csv with header as a first entry to a python dict"""
+    keys = csv[0].split(',')
+    del csv[0]
+    dataset = list()
+    for val in csv:
+        dataset.append(
+            dict(zip(keys, val.split(',')))
+        )
+    return dataset
```

Comment:
There's no need to create a list named `dataset` and append one value at a time to it. You can use a list comprehension to create an anonymous list and return it.

```python
keys = csv[0].split(',')
del csv[0]
return [
    dict(zip(keys, val.split(',')))
    for val
    in csv
]
```